public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -402,13 +406,24 @@ const GrafanaGridItem = React.forwardRef<HTMLDivElement, GrafanaGridItemProps>((
 
   // props.children[0] is our main children. RGL adds the drag handle at props.children[1]
   return (
-    <div {...divProps} style={{ ...divProps.style }} ref={ref}>
+    <div
+      {...divProps}
+      style={{ ...divProps.style }}
+      onFocus={() => panelAttentionService.setPanelWithAttention(getRefCurrent(ref))}
+      onMouseMove={() => panelAttentionService.setPanelWithAttention(getRefCurrent(ref))}
```

Comment:
This will cause a lot of function calls. I had previously prevented it by using `hasAttention`, but the service isn't reactive, so it may require more work. Maybe debouncing would help a little? This function call only sets a property on the service.